pages/Thread.html

```diff
@@ -0,0 +1,260 @@
+                <li class="dropdown">
+                    <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+                        <i class="fa fa-user fa-fw"></i> <i class="fa fa-caret-down"></i>
+                    </a>
+                    <ul class="dropdown-menu dropdown-user">
+                        <li><a href="#"><i class="fa fa-user fa-fw"></i> User Profile</a>
+                        </li>
+                        <li><a href="#"><i class="fa fa-gear fa-fw"></i> Settings</a>
+                        </li>
+                        <li class="divider"></li>
+                        <li><a href="#"><i class="fa fa-sign-out fa-fw"></i> Logout</a>
+                        </li>
+                    </ul>
+                    <!-- /.dropdown-user -->
+                </li>
+                <!-- /.dropdown -->
+            </ul>
+            <!-- /.navbar-top-links -->
+
+            <div class="navbar-default sidebar" role="navigation">
+                <div class="sidebar-nav navbar-collapse">
+                    <ul class="nav" id="side-menu">
+                        <li class="sidebar-search">
+                            <div class="input-group custom-search-form">
+                                <input type="text" class="form-control" placeholder="Search...">
+                                <span class="input-group-btn">
+                                <button class="btn btn-default" type="button">
+                                    <i class="fa fa-search"></i>
+                                </button>
+                            </span>
+                            </div>
+                            <!-- /input-group -->
+                        </li>
+                        <li class="space text-center">				<!--Added the li's after the search box-->
+                            <h4>Most Discussed Topics<br><br></h4>
+                        </li>
+                        <li class="text-center relative">
+                            <a href="1.html">1</a>
+                        </li>
+                        <li class="text-center relative">
+                            <a href="2.html">2</a>
+                        </li>
+                        <li class="text-center relative">
+                            <a href="3.html">3</a>
+                        </li>
+                        <li class="text-center relative">
+                            <a href="4.html">4</a>
+                        </li>
+                        <li class="text-center relative">
+                            <a href="6.html">5</a>
+                        </li>
+                        <li class="text-center relative">
+                            <a href="6.html">6</a>
+                        </li>
+                    </ul>
+                </div>
+                <!-- /.sidebar-collapse -->
+            </div>
+            <!-- /.navbar-static-side -->
+        </nav>
+
+
+
+
```

Comment:
Uniform indentation improves code readability.Try to have uniform indentation. Remove all unnecessary line breaks.